File: 06. Perulangan/E. Dua Pangkat/e_dua_pangkat.java
import java.util.Scanner;

class e_dua_pangkat {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        scan.close();
        while (N % 2 == 0) {
            N /= 2;
        }
        if (N == 1) {
            System.out.println("ya");
        } else {
            System.out.println("bukan");
        }
    }
}

File: 06. Perulangan/E. Dua Pangkat/e_dua_pangkat.go
package main

import (
  "fmt"
)

func main() {
  var N int32
  fmt.Scan(&N)
  for {
    if (N % 2) == 0 {
      N = N / 2
    } else {
      break
    }
  }
  if N == 1 {
    fmt.Println("ya")
  } else {
    fmt.Println("bukan")
  }
}
